.github/workflows/test.yml

```diff
@@ -32,7 +34,21 @@ jobs:
         run: go build -v ./...
 
       - name: Build X
-        run: go build -C x -o bin/ -v ./...
+        run: go build -C x -o "${{ env.OUTPUT_DIR }}/" -v ./...
+      
+      - name: Build Go Mobile
+        if: matrix.os == 'macos-latest' || matrix.os == 'ubuntu-latest'
+        run: go build -C x -o "${{ env.OUTPUT_DIR }}/" golang.org/x/mobile/cmd/gomobile golang.org/x/mobile/cmd/gobind
+
+      - name: Build Mobileproxy (Android)
+        if: matrix.os == 'ubuntu-latest'
+        run: PATH="${{ env.OUTPUT_DIR }}:$PATH" gomobile bind -ldflags='-s -w' -v -target=android -androidapi=21 -o "${{ env.OUTPUT_DIR }}/mobileproxy.aar" github.com/Jigsaw-Code/outline-sdk/x/mobileproxy
```

Comment:
Does this path (`github.com/Jigsaw-Code/outline-sdk/x/mobileproxy`) work for PR? Will `gomobile` always download the latest package from GitHub?

Reply:
The module name is specified in the go.mod file, so Go knows it's the local package.
I demonstrated that by introducing a compile error locally and showing it breaks.